store/src/sqlite/model.rs

```diff
@@ -47,6 +47,11 @@ where
         Ok(store)
     }
 
+    /// Access the underlying sqlite database pool.
+    pub fn pool(&self) -> &SqlitePool {
```

Comment:
What is this for? I didn't find the usages.

Reply:
Good catch, will delete. I added it thinking I was going to implement the new Access traits in api, and would need access to the pool.. and realized it was a really bad idea to expose sqlx stuff there and forgot to delete it.